### PR TITLE
Improve documentation of version 16 changes

### PR DESCRIPTION
**Pull request by david-a-wheeler**
This uses the email from @expln on Aug 8, 2023.

### CODE REVIEW

**Comment by david-a-wheeler** [comment]
@expln - I hope this is correct!

**Review by expln — APPROVED** [review]
Yes, everything is correct. I suggested only one change. Thanks!

File: docs/index.md [hunk 1]
```diff
@@ -4289,11 +4289,42 @@ reverse chronological order. We'll emphasize user-visible changes.
...
+* [Issue 82](https://github.com/expln/metamath-lamp/issues/82) Replacement fields should use selections consistently
+* [Issue 11](https://github.com/expln/metamath-lamp/issues/11) When the goal statement is proved then the completed proof is shown automatically.
+* [Issue 47](https://github.com/expln/metamath-lamp/issues/47) Automate merging of statements (see more details below)
+* [Issue 149](https://github.com/expln/metamath-lamp/issues/149) Allow merging multiple statements with one merge button click
+* The Ability to specify number of assertions per page on the
+  proof explorer tab.
+* The algorithm for creating proof tables was changed a bit, so now
+  it should produce same proof tables as on metamath.org. (If you
+  find a discrepancy between proofs tables on metamath.org and mm-lamp
+  for the same proof, then please feel free to let me know.)
```
Comment (expln) [inline docs/index.md:4305]: `please feel free to let me know` - I think this should be changed: `please feel free to file an issue in the mm-lamp repository - https://github.com/expln/metamath-lamp/issues`

**Review by expln — APPROVED** [review]

File: docs/index.md (hunk 1 above)
Comment (david-a-wheeler) [reply docs/index.md:4305]: 100% agree, that's much better. I'm fixing it right now.

**Pull request merged**